plugins/backend/backstage-plugin-argo-cd-backend/src/service/argocd.service.ts

```diff
@@ -83,15 +85,12 @@ export class ArgoService implements ArgoServiceApi {
   async getRevisionData(
```

Comment:
this api does not support selector.
<img width="1625" alt="image" src="https://github.com/RoadieHQ/roadie-backstage-plugins/assets/62197019/e09cfe15-2ced-4c4c-b791-e7bba46c514b">